scripts/start/projects/paper_MitiConsv.R

```diff
@@ -0,0 +1,158 @@
+
+# Load start functions
+source("scripts/start_functions.R")
+
+# ====================
+# Calibration
+# ====================
+
+source("config/default.cfg")
+
+cfg$title <- "calib_run"
+cfg$output <- c("rds_report", "validation_short")
+cfg$force_replace <- TRUE
+
+cfg$recalibrate <- TRUE
+cfg$recalibrate_landconversion_cost <- TRUE
+
+cfg$best_calib <- TRUE
+cfg$best_calib_landconversion_cost <- FALSE
+
+cfg$gms$s14_use_yield_calib <- 1
+
+# which input data sets should be used?
+# cfg$input["cellular"] <- "..."
+
+# cc is new default
+cfg <- setScenario(cfg, c("nocc_hist", "NPI", "ForestryExo"))
+cfg <- setScenario(cfg, c("MitiConsv"), scenario_config = "config/projects/scenario_config_miti_consv.csv")
+
+# sticky
+cfg$gms$factor_costs <- "sticky_feb18"
+
+# SNV habitat defintion
+cfg$gms$land_snv <- "secdforest, other"
+
+# marginal land scenario
+cfg$gms$c29_marginal_land <- "q33_marginal"
+
+start_run(cfg = cfg)
+calib_tgz <- magpie4::submitCalibration(paste(rev, "MitiConsv", sep = "_"))
+
+# ====================
+# Scenario runs
+# ====================
+
+prefix <- paste(rev, "MitiConsv", cres, sep = "_")
+
+# scenarios <- "SSP2"
+
+scenarios <- c(
+  "SSP2-REF", "SSP2-PB650-PriceAff", "SSP2-PB650-PriceProt",
+  "SSP2-PB1000-PriceAff", "SSP2-PB1000-PriceProt",
+  "SSP2-PB650-PriceAff-30by30", "SSP2-PB650-PriceProt-30by30",
+  "SSP2-PB650-PriceAff-BH", "SSP2-PB650-PriceProt-BH",
+  "SSP2-PB1000-PriceAff-30by30", "SSP2-PB1000-PriceProt-30by30",
+  "SSP2-PB1000-PriceAff-BH", "SSP2-PB1000-PriceProt-BH",
+)
+
+for (scen in scenarios) {
+  scen <- unlist(strsplit(scen, "-"))
+  ssp <- scen[grepl("SSP", scen)]
+
+  if (length(ssp) == 0 || ssp == "SSP2") {
+    ssp <- "SSP2"
+  }
+
+  source("config/default.cfg")
+
+  cfg$qos <- "short_highMem"
+
+  cfg$results_folder <- "output/:title::date:"
+
+  cfg$output <- c(
+      "output_check", "extra/disaggregation", "rds_report", "extra/runSEALSallocation"
+  )
+
+  # cfg$input["cellular"] <- "..."
+
+  cfg$input["calibration"] <- calib_tgz
+  # cfg$input["calibration"] <- "calibration_rev9_healthyLscps_25Sep24.tgz"
+
+  # cfg$gms$c_timesteps <- "5year2050"
+
+  # Climate change switched off for these runs
+  cfg <- setScenario(cfg, c(ssp, "nocc_hist", "NPI", "ForestryExo"))
+  cfg <- setScenario(cfg, c("MitiConsv"), scenario_config = "config/projects/scenario_config_miti_consv.csv")
+
+  # sticky
+  cfg$gms$factor_costs <- "sticky_feb18"
+
+  # SNV habitat defintion
+  cfg$gms$land_snv <- "secdforest, other"
+
+  cfg$gms$c56_pollutant_prices <- "coupling"
+  cfg$gms$c60_2ndgen_biodem <- "/p/projects/magpie/users/vjeetze/magpie/projects/MitiConsv/C_MitiConsv_Dec24/remind/output/C_rev5_MitiConsv_SSP2-NPi2025-rem-12/REMIND_generic_C_rev5_MitiConsv_SSP2-NPi2025-rem-12.mif"
+  cfg$path_to_report_bioenergy <- "/p/projects/magpie/users/vjeetze/magpie/projects/MitiConsv/C_MitiConsv_Dec24/remind/output/C_rev5_MitiConsv_SSP2-NPi2025-rem-12/REMIND_generic_C_rev5_MitiConsv_SSP2-NPi2025-rem-12.mif"
```

Comment:
This is the same path twice, right? Is that intended? Could also do:
```suggestion
  cfg$path_to_report_bioenergy <- cfg$gms$c60_2ndgen_biodem
```
Is it necessary to have two variables holding the same value?

Reply:
(same thing for analogue parts below)

Reply:
good point, thanks!